core/dbt/version.py

```diff
@@ -44,16 +52,40 @@ def get_version_information():
 
     plugin_version_msg = "Plugins:\n"
     for plugin_name, version in _get_dbt_plugins_info():
-        plugin_version_msg += '  - {plugin_name}: {version}\n'.format(
-            plugin_name=plugin_name, version=version
-        )
+        plugin_version = dbt.semver.VersionSpecifier.from_version_string(version)
+        latest_plugin_version = get_latest_version(version_url=get_package_pypi_url(plugin_name))
+        plugin_update_msg = ''
+        if installed == plugin_version or (
+            latest_plugin_version and plugin_version == latest_plugin_version
+        ):
+            compatibility_msg = green('Up to date!')
+        else:
+            if latest_plugin_version:
+                if installed.major == plugin_version.major:
```

Comment:
`dbt-core` + plugins are compatible at the minor-version level. So I think this might want to be:
```suggestion
                if installed.minor == plugin_version.minor:
```
If `installed.minor != plugin_version.minor`, we'd want to raise `red('Not compatible!')`